command/ibft/ibft_candidates.go

```diff
@@ -0,0 +1,94 @@
+package ibft
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-sdk/command/helper"
+	ibftOp "github.com/0xPolygon/polygon-sdk/consensus/ibft/proto"
+	empty "google.golang.org/protobuf/types/known/emptypb"
+)
+
+// IbftCandidates is the command to query the snapshot
+type IbftCandidates struct {
+	helper.Meta
+}
+
+// GetHelperText returns a simple description of the command
+func (p *IbftCandidates) GetHelperText() string {
+	return "Queries the current set of proposed candidates, as well as candidates that have not been included yet"
+}
+
+func (p *IbftCandidates) GetBaseCommand() string {
+	return "ibft candidates"
+}
+
+// Help implements the cli.IbftCandidates interface
+func (p *IbftCandidates) Help() string {
+	p.Meta.DefineFlags()
+
+	return helper.GenerateHelp(p.Synopsis(), helper.GenerateUsage(p.GetBaseCommand(), p.FlagMap), p.FlagMap)
+}
+
+// Synopsis implements the cli.IbftCandidates interface
+func (p *IbftCandidates) Synopsis() string {
+	return p.GetHelperText()
+}
+
+// Run implements the cli.IbftCandidates interface
+func (p *IbftCandidates) Run(args []string) int {
```

Comment:
I'm not sure if receiver name "p" is good in this context.

Reply:
Fixed 👍 

fef1864